docs/developer-guildelines.md

```diff
@@ -0,0 +1,17 @@
+## Contributing to the existing actions
+
+1. Fork the repo and pull it to the local master branch.
+
+1. Run __“npm install”__ in the terminal within the repo. This command downloads dependencies defined in a __package. json__ file and generates a node_modules folder with the installed modules.
```

Comment:
Before running `npm install` mention the minimum required versions of node and npm to be present in the machine